src/vs/workbench/contrib/positronPlots/browser/webviewPlotClient.ts

```diff
@@ -65,7 +65,7 @@ export abstract class WebviewPlotClient extends Disposable implements IPositronP
 	}
 
 	/** Whether the plot's underlying webview is active. */
-	isActive(): this is { _webview: { value: IOverlayWebview } } {
```

Comment:
Given that the type predicate didn't work in release builds, I simplified this and replaced its internal usage with a direct check so we don't need type assertions.